helpdesk_mgmt/security/ir.model.access.csv

```diff
@@ -2,7 +2,7 @@ id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_helpdesk_ticket_manager,helpdesk.ticket.manager,model_helpdesk_ticket,group_helpdesk_manager,1,1,1,1
 access_helpdesk_ticket_user,helpdesk.ticket.user,model_helpdesk_ticket,group_helpdesk_user,1,1,1,0
 access_helpdesk_ticket_user_personal,helpdesk.ticket.user.personal,model_helpdesk_ticket,group_helpdesk_user_own,1,1,1,0
-access_helpdesk_ticket_portal,helpdesk.ticket.portal,model_helpdesk_ticket,base.group_portal,1,0,0,0
+access_helpdesk_ticket_portal,helpdesk.ticket.portal,model_helpdesk_ticket,base.group_portal,1,1,0,0
```

Comment:
this goes very far, as external users can do all kinds of mischief now by crafting json requests that to a write on tickets. Better override [`ir.attachment#check`](https://github.com/OCA/OCB/blob/12.0/odoo/addons/base/models/ir_attachment.py#L338) to treat helpdesk tickets differently for portal users.

Reply:
Done, thanks